pandas/core/frame.py

```diff
@@ -4411,9 +4411,13 @@ def _get_agg_axis(self, axis_num):
 
     def mode(self, axis=0, numeric_only=False):
         """
-        Gets the mode of each element along the axis selected. Empty if nothing
+        Gets the mode(s) of each element along the axis selected. Empty if nothing
         has 2+ occurrences. Adds a row for each mode per label, fills in gaps
-        with nan.
+        with nan. Note that there could be multiple values returned for the selected
+        axis (when more than one item share the maximum frequency), which is the 
+        reason why a dataframe is returned. This means that if you want to impute 
+        missing values with the mode in a dataframe df, you can just do this: 
+        df.fillna(df.mode().ix[0])
```

Comment:
two things:

1. If you "quote" code with two backticks `` on each time it will be formatted as code in the online documentation.
2. I would suggest using `.iloc` instead of `.ix`, because the later will fail if the index consists of integers.